scripts/globals/weaponskills/blade_ku.lua

```diff
@@ -23,27 +23,29 @@ require("scripts/globals/weaponskills")
 local weaponskill_object = {}
 
 weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary, action, taChar)
-
     local params = {}
     params.numHits = 5
-    params.ftp100 = 1 params.ftp200 = 1 params.ftp300 = 1
-    params.str_wsc = 0.1 params.dex_wsc = 0.1 params.vit_wsc = 0.0 params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.0 params.chr_wsc = 0.0
+    params.ftp100 = 1.0 params.ftp200 = 1.0 params.ftp300 = 1.0
+    params.str_wsc = 0.1 params.dex_wsc = 0.1 params.vit_wsc = 0.0
+    params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.0
+    params.chr_wsc = 0.0
     params.crit100 = 0.0 params.crit200 = 0.0 params.crit300 = 0.0
     params.canCrit = false
-    params.acc100 = 0.9 params.acc200= 1 params.acc300= 1.1
-    --  the TP bonus for TP 1,000 and above has been raised, which has substantially raised the hit rate http://wiki.ffo.jp/html/732.html
-    -- data on ws accuracy are difficult to come by, would need to decide on sane value for the accuracy boost stated in JPWiki for now I'm only adding 0.1 per tier
-    params.atk100 = 1; params.atk200 = 1; params.atk300 = 1
+    -- Sufficient data for ACC bonus/penalty does not exist; assuming no penalty and 10% increase per 1,000 TP
+    -- http://wiki.ffo.jp/html/732.html does not list ACC Bonus
+    -- https://www.bg-wiki.com/ffxi/Blade:_Ku does not list ACC Bonus
+    params.acc100 = 1.0 params.acc200 = 1.1 params.acc300 = 1.2
```

Comment:
We've been over this in the past and it really boils down to we can't really know what these values are. https://forum.square-enix.com/ffxi/threads/42615

It looks like based on the text in the linked article that the acc bonus for 1k-2k tp is stronger than 2k-3k, if I understand it right. What should the values be? I cant really say. is 10% and 20% acc bonus reasonable for this? In my opinion, probably. If anything, 1.15 to 2k tp then 1.2 to 3k if we want to guess. (I would rather not, but it is hard to quantify without explicitly going out of your way to test it specifically).

But this choice isn't just up to me. Anyone else want to chime in?

Reply:
Would the safer bet here be to do 5% increases? 1.0/1.05/1.10? A 20% ACC bonus is significant at 1,000 base ACC.

Reply:
I'm going to revisit "accuracy varies with tp" later in/after damage overhaul to lua, so letting this slide